lib/src/thread.c

```diff
@@ -251,7 +251,7 @@ static void set_timeout(struct timespec *timeout, uint64_t ms_from_now)
 }
 #endif
 
-#if !__APPLE__
+#if !__APPLE__ && !__SWITCH__
```

Comment:
pthread_clockjoin_np is not avaliable in aarch64_none_elf.

Reply:
it can be removed for switch, it's just used for the discover command in the cli